constitution.tex

```diff
@@ -770,6 +770,9 @@
 \item Candidates must be Active Members
 \end{enumerate}
 
+\asubsection{Overriding Directorships Qualifications}
+Qualifications pertaining to a specific directorship election can be overridden via a Constitutional Override, as explained in \ref{Semantic Changes}. Overrides will apply to a specific qualification of the directorship, and if passed, will be overridden for all candidates for said directorship.
```

Comment:
Please comply to the style conventions https://github.com/ComputerScienceHouse/Constitution#document-style